tasklib/backends.py

```diff
@@ -427,3 +427,34 @@ def valid(output):
 
     def sync(self):
         self.execute_command(['sync'])
+
+    def _get_task_attrs(self):
+        TASK_STANDARD_ATTRS = [
+            'annotations',
+            'entry',
+            'depends',
+            'description',
+            'due',
+            'end',
+            'imask',
+            'mask',
+            'modified',
+            'parent',
+            'priority',
+            'project',
+            'recur',
+            'scheduled',
+            'start',
+            'status',
+            'tags',
+            'until',
+            'uuid',
+            'wait',
+        ]
+        available_task_attrs = TASK_STANDARD_ATTRS
+        udas = set()
+        for index in self.config:
+            if 'uda' in index:
```

Comment:
Should this not check for `index.startswith('uda.')`?

Reply:
Yup, fixed in f055fce @robgolding